pkg/kapp/resourcesmisc/kappctrl_k14s_io_v1alpha1_app_test.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package resourcesmisc_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	ctlres "github.com/vmware-tanzu/carvel-kapp/pkg/kapp/resources"
+	ctlresm "github.com/vmware-tanzu/carvel-kapp/pkg/kapp/resourcesmisc"
+)
+
+func TestKappctrlK14sIoV1alphaAppFailure(t *testing.T) {
```

Comment:
nit: Should we omit `K14sIo`?
`V1alpha` -> `V1alpha1`

Reply:
It should be `V1alpha1`

I am maintaining how we name tests after GVK for all resources. (Taken directly from the associated struct name) 🤔